STYLE_GUIDE.md

```diff
@@ -118,6 +128,22 @@ A procedure is a series of numbered steps that a user follows to complete a spec
 - Parallel language constructions
 - Consistent formatting
 
+Replace pointer-specific language with device-agnostic language to accommodate readers with disabilities and users of various input methods and devices, including the pointer, keyboard, and touch screens.
+
+For example, instead of the term *click*, which is pointer-specific, use *choose*, which is more generic and device-agnostic. However, when the generic language makes it difficult to understand the instructions (for example, in the case of opening a context menu), you can include pointer-specific hints. Use your judgment. If you have a question, ask your editor.
+
+Use the following language to describe UI interactions:
+
+- Use *choose* to describe moving to a UI component such as a tab or pane or taking action on a button or menu.
+
+- Use *select* to describe picking a user-specific resource or enabling one of several options. Contrast with *clear* to turn off previously selected options.
```

Comment:
Do we use *select* only when there are options that can be "undone"? So, for a checkbox group or radio button group we use *select*, but for a dropdown we use *choose*? The distinction between *choose* and *select* is not 100% clear to me.
Also, if we use *select*, do we specify what UI component to select by label or type? So, "Select the Modify button." or "Select Modify"?

Reply:
Added a table with examples to clarify.